Source/Applications/SystemCenter/wwwroot/Scripts/TSX/SystemCenter/RemoteXDA/SystemSettingsTab.tsx

```diff
@@ -68,15 +68,15 @@ const SystemSettingsTab = (props: IProps) => {
     }
 
     return (
-        <div className="card" style={{ marginBottom: 10, maxHeight: window.innerHeight - 215 }}>
+        <div className="card" style={{ flex: 1, display: 'flex', flexDirection: 'column', overflow: 'hidden' }}>
             <div className="card-header">
                 <div className="row">
                     <div className="col">
                         <h4>Remote openXDA Instance Connection Information:</h4>
                     </div>
                 </div>
             </div>
-            <div className="card-body" style={{ maxHeight: window.innerHeight - 315, overflowY: 'auto' }}>
+            <div className="card-body" style={{ flex: 1, overflowY: 'auto' }}>
```

Comment:
overflow here should be hidden. Overflowing should be handled by child table.

Reply:
There is no child table for this tab